pyopenms_viz/_plotly/core.py

```
@@ -28,6 +28,7 @@
 from .._config import bokeh_line_dash_mapper
 from .._misc import ColorGenerator, MarkerShapeGenerator, is_latex_formatted
 from ..constants import PEAK_BOUNDARY_ICON, FEATURE_BOUNDARY_ICON
+from .._dataframe import DataFrameWrapper
```

Ruff (0.8.2)

`.._dataframe.DataFrameWrapper` imported but unused

Remove unused import: `.._dataframe.DataFrameWrapper`

(F401)